textdescriptives/components/coherence.py

```diff
@@ -0,0 +1,80 @@
+from typing import List
+
+import numpy as np
+from spacy.language import Language
+from spacy.tokens import Doc
+
+
+@Language.factory("textdescriptives.coherence")
+def create_coherence_component(nlp: Language, name: str):
```

Comment:
Should we allow for an argument e.g.
```
order: Tuple[int, int] = (1, 2): The order of coherence is defined as a tuple of the lowest to the highest desired order. Default to (1, 2). Indicating First and second-order coherence. 
```

Reply:
From Alberto's paper it seems that only first and second order are used in practice. Not sure it's worth adding the extra naming logic to the class